app/routes/routes.py

```diff
@@ -0,0 +1,232 @@
+# from flask import Blueprint, jsonify, request, abort, make_response
```

Comment:
You can probably delete this file.

Reply:
Yes! I only kept it for my own study habits, so I can review what I did//how I improved! Otherwise, yes I would have deleted it!